salt/beacons/status.py

```diff
@@ -116,9 +117,11 @@ def beacon(config):
             'time': 'all',
         }
 
-    ret = {}
     for func in config:
-        data = __salt__['status.{0}'.format(func)]()
+        try:
+            data = __salt__['status.{0}'.format(func)]()
+        except (KeyError, AttributeError):
```

Comment:
A `KeyError` makes sense but I'm not sure why an `AttributeError` would be raised here?